src/Base64.test.tsx

```diff
@@ -0,0 +1,16 @@
+import Base64 from "./Base64";
+import {shallow} from 'enzyme';
+
```

Comment:
There are a lot of eslint typescript warnings here. Can we use the @types/enzyme npm package?

Reply:
Done